data.proto

```diff
@@ -49,6 +49,7 @@ message AgentConfig {
   string model_id = 3;
   int32 model_version = 4;
   int32 actor_index = 5; // Used to figure out if an agent is the current_player in the observation space
+  string device = 6;
```

Comment:
Still not a big fan of having that there, is there a use case for having some actors in a run using cpu and some others using gpu? If not, shouldn't that be a part of the `torch_agents` service configuration?